### PR TITLE
Mesos tls kerberos 4.0.0 post

### PR DESCRIPTION
**Pull request by cyan-confluent**

### CODE REVIEW

**Review by maxzheng — APPROVED** [review]
LGTM. Perhaps wait for @aayars to double check

**Comment by cyan-confluent** [comment]
[clabot:check]

File: debian/base/include/etc/confluent/docker/mesos-setup.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        keytool -import \
+                -trustcacerts \
+                -alias root \
+                -file $MESOS_SANDBOX/.ssl/ca-bundle.crt \
+                -storepass changeit \
+                -keystore /tmp/kafka-truststore.jks -noprompt
+
+
+        if [ "$KERBEROS_ENABLED" = "true" ]; then
```
Comment (cyan-confluent) [inline debian/base/include/etc/confluent/docker/mesos-setup.sh]: Now that I think about it. I'm not sure if we really want this line. This line provides some convenience to the user, but if the user is already configuring secrets, he might as well be aware of configuring KAFKAREST_OPTS.

File: debian/base/include/etc/confluent/docker/mesos-setup.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [ -z $SKIP_MESOS_AUTO_SETUP ]; then
```
Comment (aayars) [inline debian/base/include/etc/confluent/docker/mesos-setup.sh]: Can this check for a positive condition (like ENABLE_MESOS) instead?

Comment (cyan-confluent) [reply debian/base/include/etc/confluent/docker/mesos-setup.sh]: Totally understand positives are preferred when coding. I'm making sure users who choose to skip this really knows what they are doing. They should be using the default setup unless they really really want to manually configure.

**Review by aayars — APPROVED** [review]
LGTM.

We're in the process of moving to new repos for all of these images, and will need to make this change there too. The new base won't be used for this release, though.

**Pull request merged**